src/components/BulkOperationDetailsPage/BulkOperationDetails.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import 'moment-timezone';
+import PropTypes from 'prop-types';
+import {
+  Button, DataTable, useToggle, FullscreenModal, Stack,
+} from '@openedx/paragon';
+
+import { formatDateTime } from '../../utils';
+import './styles.scss';
+
+const ImageCell = ({ value }) => (
+  value ? (
+    <a href={value} target="_blank" rel="noopener noreferrer">
+      View Image
+    </a>
+  ) : null
+);
+
+ImageCell.propTypes = {
+  value: PropTypes.string,
+};
+
+ImageCell.defaultProps = {
+  value: null,
+};
+
+const parseCSV = async (url) => {
+  const response = await fetch(url);
+  const text = await response.text();
+  const lines = text.trim().split('\n');
+
+  const headers = lines[0].split(/,(?=(?:(?:[^"]*"){2})*[^"]*$)/);
```

Comment:
ah, what's the reason for using this regex?

Reply:
To split a CSV line correctly on commas, only those commas that are not inside double quotes, I added this regex to fix issues I was having with columns being split incorrectly.